class-one-exchange-htb.js

```diff
@@ -283,10 +317,10 @@ function ClassOneExchangeHtb(configs) {
             /* Using the above variable, curBid, extract various information about the bid and assign it to
             * these local variables */
 
-            var bidPrice = curBid.price; /* the bid price for the given slot */
-            var bidSize = [curBid.width, curBid.height]; /* the size of the given slot */
-            var bidCreative = curBid.adm; /* the creative/adm for the given slot that will be rendered if is the winner. */
-            var bidDealId = curBid.dealid; /* the dealId if applicable for this slot. */
+            var bidPrice = curBid.cpm;
+            var bidSize = [curBid.width, curBid.height];
+            var bidCreative = curBid.ad;
+            var bidDealId = null; // do not have dealId in our bid response
```

Comment:
We do not have deal Id params for requests to our endpoint, neither for responses from our bidder. So I set it as null here. They have test cases for responses with deal Id so I didn't remove some codes for deal Id here.